src/System.Reflection.Metadata/src/Resources/Strings.resx

```diff
@@ -354,4 +354,7 @@
   <data name="NotTypeDefOrRefOrSpecHandle" xml:space="preserve">
     <value>Specified handle is not a TypeDefinitionHandle, TypeRefererenceHandle, or TypeSpecificationHandle.</value>
   </data>
+  <data name="NotCodeViewEntry" xml:space="preserve">
+    <value>The Debug Directory was not a code view.</value>
```

Comment:
Why is "Directory" capitalized?  This should be "The Debug directory was not a code view."